doc/userguide/performance/high-performance-config.rst

```diff
@@ -1,19 +1,341 @@
+
+In the case below we are using 36 worker threads out of NUMA node 1's CPU, af-packet runmode with `cluster-type: cluster_qm`.
+
+If the CPU's NUMA set up is as follows:
+
+::
+
+    lscpu
+    Architecture:        x86_64
+    CPU op-mode(s):      32-bit, 64-bit
+    Byte Order:          Little Endian
+    CPU(s):              72
+    On-line CPU(s) list: 0-71
+    Thread(s) per core:  2
+    Core(s) per socket:  18
+    Socket(s):           2
+    NUMA node(s):        2
+    Vendor ID:           GenuineIntel
+    CPU family:          6
+    Model:               79
+    Model name:          Intel(R) Xeon(R) CPU E5-2697 v4 @ 2.30GHz
+    Stepping:            1
+    CPU MHz:             1199.724
+    CPU max MHz:         3600.0000
+    CPU min MHz:         1200.0000
+    BogoMIPS:            4589.92
+    Virtualization:      VT-x
+    L1d cache:           32K
+    L1i cache:           32K
+    L2 cache:            256K
+    L3 cache:            46080K
+    NUMA node0 CPU(s):   0-17,36-53
+    NUMA node1 CPU(s):   18-35,54-71
+
+It is recommended that 36 worker threads are used and the NIC set up could be as follows:
+
+::
+
+    rmmod i40e && modprobe i40e
+    ifconfig eth1 down
+    /usr/local/sbin/ethtool -L eth1 combined 36
+    /usr/local/sbin/ethtool -K eth1 rxhash on
+    /usr/local/sbin/ethtool -K eth1 ntuple on
+    ifconfig eth1 up
+    ./set_irq_affinity local eth1
+    /usr/local/sbin/ethtool -X eth1 hkey 6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A equal 36
+    /usr/local/sbin/ethtool -A eth1 rx off tx off
+    /usr/local/sbin/ethtool -C eth1 adaptive-rx off adaptive-tx off rx-usecs 125
+    /usr/local/sbin/ethtool -G eth1 rx 1024
+    for proto in tcp4 udp4 tcp6 udp6; do
+        echo "/usr/local/sbin/ethtool -N eth1 rx-flow-hash $proto sdfn"
+        /usr/local/sbin/ethtool -N eth1 rx-flow-hash $proto sdfn
+    done
+
+In the example above the `set_irq_affinity` script is used form the NIC drivers sources.
+In the cpu affinity section of suricata.yaml config :
+
+::
+
+ # Suricata is multi-threaded. Here the threading can be influenced.
+ threading:
+  set-cpu-affinity: yes
+  # Tune cpu affinity of threads. Each family of threads can be bound
+  # on specific CPUs.
+  #
+  # These 2 apply to the all runmodes:
+  # management-cpu-set is used for flow timeout handling, counters
+  # worker-cpu-set is used for 'worker' threads
+  #
+  # Additionally, for autofp these apply:
+  # receive-cpu-set is used for capture threads
+  # verdict-cpu-set is used for IPS verdict threads
+  #
+  cpu-affinity:
+    - management-cpu-set:
+        cpu: [ "1-10" ]  # include only these CPUs in affinity settings
+    - receive-cpu-set:
```

Comment:
this is not necessary in that case right? won't hurt either.
Also would remove the comments.